Packs/McAfeeNSM/Integrations/McAfeeNSMv2/McAfeeNSMv2.py

```diff
@@ -588,6 +744,69 @@ def create_body_create_rule(rule_type: str, address: List, number: int,
         }
 
 
+def create_body_create_rule_for_v10(rule_type: str, address: List, number: int,
+                                    from_to_list: list[dict[str, Any | None]], state: str = "Enabled") -> tuple:
+    """ create part of the body for the command create/update_rule_object for v10
+        Args:
+            rule_type: str - The type of the rule.
+            address: List - A list of addresses, if relevant.
+            number: int - The number of the IPV.
+            from_to_list: List - A list that contains dictionaries with from and do addresses.
+            stste: str - An Enabled or Disabled state.
+        Returns:
+            Returns the body for the request.
+        """
+    # build a list of dictionaries with the state and the address
+    list_to_send: list[dict] = [
+        {"value": single_address, "state": STATE_TO_NUMBER.get(state)}
+        for single_address in address]
+    # for parameters with a range, we need to add the state to the dictionary
+    if from_to_list:
+        from_to_list[0].update({"state": STATE_TO_NUMBER.get(state)})
+
+    if 'HOST' in rule_type:
```

Comment:
Both 'HOST' and 'ADDRESS_RANGE' should be consts.

Reply:
The same